js/script.js

```diff
@@ -13,5 +13,216 @@ function closeMenu() {
+        and scrambled it 1960s with the releaLorem Ipsum is simply dummy
+        text of the printing and typesetting industry. Lorem Ipsum has been
+        the industry's standard dummy text ever since the 1500s, when an
+        unknown printer took a galley of type and scrambled it 1960s with
+        the relea</p
+      >
+    </div>
+    <div class="btnPopup"
+      ><a href="#" id="livePopup" class="btn-see-project"
+        >See Live
+        <ion-icon class="ion-icon" name="caret-forward-circle-outline"></ion-icon></a
+      ><a href="#" id="sourcePopup" class="btn-see-project"
+        >See Source <ion-icon class="ion-icon" name="logo-github"></ion-icon></a
+    ></div>
+  `;
+mainPopup.appendChild(popup);
+
+function openProject() {
+  main.appendChild(mainPopup);
+  const closePopup = document.querySelector('#close-btn1');
+  closePopup.addEventListener('click', () => {
+    main.removeChild(mainPopup);
+  });
+}
+
+/// // *** POPUP WINDOW desktop *** /////
+
+const mainPopupDesktop = document.createElement('div');
+mainPopupDesktop.className = 'mainPopupDesktop';
+mainPopupDesktop.innerHTML = `<div class="popupDesktop">
+<div class="imagePopupDesktop">
+  <button class="exitBtn"></button>
+  <img src="../image/SnapshootPortfolioDesktop.jpg" alt="Project image" />
+</div>
+<div class="components">
+  <h2>Keeping track of hundreds of components</h2>
+  <div class="btnPopup"
+    ><a href="#" id="livePopup" class="btn-see-project"
+      >See Live
+      <ion-icon name="caret-forward-circle-outline"></ion-icon></a
+    ><a href="#" id="sourcePopup" class="btn-see-project"
+      >See Source <ion-icon name="logo-github"></ion-icon></a
+  ></div>
+</div>
+<div class="flex-container-div">
+  <ul class="flex-container" id="flex-container">
+    <li class="flex-items"> Codekit</li>
+    <li class="flex-items">GitHub</li>
+    <li class="flex-items">JavaScript</li>
+    <li class="flex-items">Bootstrap</li>
+    <li class="flex-items">Terminal</li>
+    <li class="flex-items">Codepen</li>
+  </ul>
+</div>
+<p class="textPopupDesktop"
+  >Lorem Ipsum is simply dummy text of the printing and typesetting
+  industry. Lorem Ipsum has been the industry's standard dummy text ever
+  since the 1500s, when an unknown printer took a galley of type and
+  scrambled it 1960s with the releaLorem Ipsum is simply dummy text of
+  the printing and typesetting industry. Lorem Ipsum has been the
+  industry's standard dummy text ever since the 1500s, when an unknown
+  printer took a galley of type and scrambled it 1960s with the relea</p
+>
+</div>`;
+
+function openProjectDesktop() {
+  main.appendChild(mainPopupDesktop);
+  const closePopupDesktop = document.querySelector('.exitBtn');
+  closePopupDesktop.addEventListener('click', () => {
+    main.removeChild(mainPopupDesktop);
+  });
+}
+
+function checkMeadiaQuery() {
+  if (window.innerWidth > 768) {
+    openProjectDesktop();
+  } else {
+    openProject();
+  }
+}
+
```

Comment:
- [ ] Kindly make sure that when a user clicks the `see project` button a popup opens with the project details
- [ ] Please ensure the button works and the details for each project are shown on the modalpopulated 

https://user-images.githubusercontent.com/69638013/203553499-f7f0eb90-cc46-4c9f-8504-a6c445b6dd85.mp4